external/common/include/cycle_counter.hpp

```diff
@@ -234,7 +234,7 @@ INLINE_ELAPSED(__inline__)
 
 /* Visual C++ (FIXME: how to detect compilation for x86-64?) */
 #if _MSC_VER >= 1400 && !defined(HAVE_TICK_COUNTER)
-typedef ULONG64 ticks;
+typedef unsigned __int64 ticks;
```

Comment:
>Was this a manual change?

Yes, `ULONG64` is defined in nowhere. It seems that is just a deprecated MSVC data type.